.github/workflows/snapshots.yml

```diff
@@ -0,0 +1,75 @@
+name: snapshots
+description: Daily uploads package snapshots to https://github.com/mlflow/mlflow/releases/tag/nightly.
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # Runs daily at midnight UTC
+  workflow_dispatch: # Allows manual triggering
+  pull_request: # To test updates
+    paths:
+      - ".github/workflows/snapshots.yml"
+      - ".github/scripts/snapshots.js"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    permissions:
+      contents: write # Needed to push tags and create releases
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: ./.github/actions/setup-python
+      - uses: ./.github/actions/setup-java
+      - uses: ./.github/actions/setup-node
+      - uses: r-lib/actions/setup-r@bd49c52ffe281809afa6f0fecbf37483c5dd0b93 # v2.11.3
+
+      - name: Create artifacts directory
+        run: |
+          ARTIFACT_DIR=$(mktemp -d)
+          echo "ARTIFACT_DIR=$ARTIFACT_DIR" >> $GITHUB_ENV
+
+      - name: Build Python packages
+        id: build-dist
+        run: |
+          pip install build
+          python dev/build.py
+          find $PWD/dist -type f -name "*.whl" -exec cp {} $ARTIFACT_DIR/ \;
+          python dev/build.py --package-type skinny
+          find $PWD/dist -type f -name "*.whl" -exec cp {} $ARTIFACT_DIR/ \;
+          ls $ARTIFACT_DIR
```

Comment:
Do we want to build tracing package?

Reply:
I don't think we need to for now. We can add it when we need to.

Reply:
Make sense